### ISSUE TITLE
Unable to delete function metadata.

### ISSUE THREAD

**Issue by mtartare** [naim94a/lumen#56]
**Describe the bug**
On IDA Pro 8.1, deleting a function metadata fails with the following error:
```
lumina: lumen: error: invalid data.
Command "LuminaFunctionsDeleteMd" failed
```
This works when trying to the the same with a private Lumina server.

**To Reproduce**
* Rename a function
* Click "Push current function metadata"
* Click "Pull current function metadata"
* Click "Delete current function metadata"

**Expected behavior**
The function's metadata is successfully deleted

**Server (please complete the following information):**
 - OS: Debian 11
 - Architecture: x86_64

**Client (please complete the following information):**
 - OS: Windows 10
 - Architecture: x86_64
 - IDA Version: 8.1

**Comment by naim94a** [comment]
Thanks for reporting. More research is required before implementing this feature.

**Comment by naim94a** [comment]
Merged in #60
